packages/SwingSet/test/devices/test-devices.js

```diff
@@ -338,6 +354,12 @@ test.serial('liveslots throws when D() gets promise', async t => {
   const hostStorage = provideHostStorage();
   await initializeSwingset(config, ['promise1'], hostStorage, t.context.data);
   const c = await makeSwingsetController(hostStorage, {});
+  for (let i = 0; i < 4; i += 1) {
+    // eslint-disable-next-line no-await-in-loop
+    await c.step(); // vat start
+    // eslint-disable-next-line no-await-in-loop
+    await c.step(); // bring out your dead
+  }
```

Comment:
ok this one really does need the careful pre-stepping

BTW feel free to hoist that `eslint-disable` to the top of this source file, that'd probably remove a lot of clutter